clang/include/clang/AST/Stmt.h

```diff
@@ -109,16 +109,27 @@ class alignas(void *) Stmt {
 
   //===--- Statement bitfields classes ---===//
 
+  enum { NumStmtBits = 10 };
+
+#define STMT(CLASS, PARENT)
+#define STMT_RANGE(BASE, FIRST, LAST)
+#define LAST_STMT_RANGE(BASE, FIRST, LAST)                                     \
+  static_assert(                                                               \
```

Comment:
suggestion: `static_assert(llvm::isInt<NumStmtBits>(StmtClass::LAST##Class), "The number of 'StmtClass'es is strictly bound by a bitfield of width NumStmtBits");`